gravity-forms/gw-gravity-forms-custom-js.php

```diff
@@ -91,12 +91,18 @@ public function noconflict_styles( $scripts = array() ) {
 
 	public function add_custom_js_setting( $settings, $form ) {
 
+		// GF 2.5 may fire `gform_form_settings` before `save_custom_js_setting`
+		$customJS = rgar( $form, 'customJS' );
+		$postJS   = esc_html( rgpost( 'custom_js' ) );
+		if ( $postJS && $postJS !== $customJS ) {
+			$customJS = $postJS;
+		}
```

Comment:
I suspect we can simplify the logic here by always using the posted JS if it's present. Was there a scenario you encountered where it didn't make sense to use the posted JS?

Reply:
That makes sense. Posted JS should always be used since it'll be either the latest version or the current version as is. Fixed :)